pkgs/applications/science/biology/neuron/default.nix

```diff
@@ -54,7 +56,7 @@ stdenv.mkDerivation rec {
   configureFlags = with stdenv.lib;
                     [ "--with-readline=${readline}" "--with-iv=${iv}" ]
                     ++  optionals (python != null)  [ "--with-nrnpython=${python.interpreter}" ]
-                    ++ (if mpi != null then ["--with-mpi" "--with-paranrn"]
+                    ++ (if useMpi then ["--with-mpi" "--with-paranrn"]
```

Comment:
```suggestion
                    ++ lib.optioanls useMpi ["--with-mpi" "--with-paranrn"]
```

Reply:
That won't work here: the `else` part is needed in case it is turned off.